src/Service/SessionFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Service;
+
+class SessionFactory
+{
+
+    /**
+     * @param string $startDate
+     * @param int $repeatWeeks
+     * @return array
+     * @throws \Exception
+     */
+    public function repeatPerWeeks(string $startDate, int $repeatWeeks): array
```

Comment:
nevisai tam factoriai naudojami. Is esmes, factoris skirtas pati objekta sukurti ir susettinti jam kintamuosius, tu konkreciai ta factorio darba atlieki kontroleryje. As manyciau, kad tu tretum cia tureti tris metodus: create, createRepeated(cia kur ne viena, o per kelias savaites create'ini), ir sitas konkretus metodas turetu buti private, kuri kviecia pirmieji